src/stampy_nlp/utilities/coda_utils.py

```diff
@@ -38,7 +40,7 @@ def get_df_data():
             url = values['Link']
 
         # add to data_list if some kinds of question, even if unanswered
-        if isinstance(pageid, str) and len(pageid) >= 4:
+        if isinstance(pageid, str) and len(pageid) >= 4 and status not in EXCLUDE_STATUS:
```

Comment:
the `isinstance` isn't needed here - if it's not a string, then line 33 would have blown up

Reply:
Oh yeah, that was leftover during the UI ID transition from int to str and there were still a few stragglers remaining. Removed it.